modules/navbar.js

```diff
@@ -0,0 +1,28 @@
+const navBar = () => `
```

Comment:
Continue in the tradition of starting module names with capital letters. This differentiates them from ordinary funtions.

Reply:
Taken